### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by ocke**
:v:

/cc @zendesk/vegemite

### Description
updated readme

### References
* JIRA: 

### Risks

### CODE REVIEW

**Comment by adammw** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+## Description
+Zendesk Apps Tools (ZAT) is the number one tool for developing Zendesk Apps. It makes it easy to develop [Zendesk Apps](http://developer.zendesk.com/documentation/apps/). We have a [guide](http://developer.zendesk.com/documentation/apps/reference/tools.html) that explains how to use this tool.
 
-# How to use
+## Owners
+This repo is owned and maintained by the Zendesk Apps team. You can reach us on vegemite@zendesk.com. We are located in Melbourne!
 
-Please refer to [our guide](http://developer.zendesk.com/documentation/apps/reference/tools.html).
+## Getting Started
+`bundle install` is really all that is needed to get it going.
```
Comment (sandlerr) [inline README.md]: misleading? users should `gem install zendesk_apps_tools` without even cloning the repo

**Comment by sandlerr** [comment]
+1

File: README.md (hunk 1 above)
Comment (ocke) [inline README.md]: Hmm, good point.

I guess we should say: in order to develop this tool you need to git clone it and bundle install. But if you just want to *use* zat, simple gem install zendesk_apps_tools?

Comment (sandlerr) [inline README.md]: I think a lot of open source projects tend to have a `Using` heading and a `Hacking` heading (which includes testing and how to contribute), we could do something along those lines.

**Pull request merged**